languages.yaml

```diff
@@ -2612,6 +2612,33 @@ TypeScript:
       - "#007ACC"
       - "#FFFFFF"
     chip: "#2B7489"
+Twig:
+  type: markup
+  ascii: |
+    {0}########################################
+    @{0}...........................:::........@{0}
+    @{0}.........................^!^:::.......@{0}
+    @{0}.......................:!7:...........@{0}
+    @{0}......................:7?:............@{0}
+    @{0}......................!J~.............@{0}
+    @{0}.....................^?7..............@{0}
+    @{0}.....................7J^..............@{0}
+    @{0}....................^JJ..::...........@{0}
+    @{0}............:.......!Y7.~:............@{0}
+    @{0}............^..::^:.75!~:.............@{0}
+    @{0}............~:...:~~?57~..............@{0}
+    @{0}............!^.....!JY7^..............@{0}
+    @{0}...........:77.....~??7^..............@{0}
+    @{0}...........:??^....~J??~..............@{0}
+    @{0}...........:77:...:~!!!~:.............@{0}
+    @{0}...........:^^:.......................@{0}
+    @{0}......................................@{0}
+    ########################################
```

Comment:
@o2sh what do you think, should these characters be explicitly styled as "white"?

Reply:
what about?
<img width="429" alt="Screenshot 2024-01-20 at 11 42 23 AM" src="https://github.com/o2sh/onefetch/assets/3926244/61de575d-41e8-4d49-89e0-7cd1851102c0">

Reply:
I meant that, instead of just `{0}` with `green`, it might be better to have two colors, `{0}` and `{1}` for `white` and `green`.